app/assets/images/payment_icons/farmlands.svg

```diff
@@ -0,0 +1 @@
4.22-1.02 1.44-.65 2.71-1.58 3.91-2.8m0 8.34c-1.07.96-2.02 1.79-2.88 2.57-.85.75-2.01 1.5-3.45 2.28-1.41.7-2.78 1.25-4.1 1.61-1.3.34-3.2.54-5.57.54-4.45 0-8.15-1.58-11.08-4.74-2.93-3.16-4.4-7.15-4.4-11.97 0-3.97.72-7.16 2.23-9.59 1.49-2.44 3.65-4.35 6.46-5.81 2.81-1.45 6.23-2.46 10.13-3.03 3.96-.57 8.22-1.01 12.78-1.32v-.29c0-2.95-1.05-5.03-3.1-6.15-2.01-1.16-5.1-1.71-9.17-1.71-1.87 0-4.02.36-6.5 1.09-2.48.73-4.9 1.66-7.27 2.83h-1.33v-12.8c1.57-.47 4.09-1.01 7.58-1.69 3.46-.65 6.97-.96 10.53-.96 8.7 0 15.05 1.53 19 4.59 3.99 3.03 5.98 7.7 5.98 13.98v36.23h-15.85v-5.66zm74.18 5.66h-15.82V338.8c0-2.15-.1-4.3-.24-6.43-.17-2.13-.5-3.71-.96-4.71-.53-1.19-1.29-2.05-2.34-2.56-1.04-.52-2.39-.78-4.13-.78-1.29 0-2.59.23-3.94.75-1.33.49-2.74 1.29-4.25 2.41v37.72h-15.85v-53.27h15.85v5.99c2.6-2.33 5.13-4.12 7.57-5.39 2.45-1.3 5.16-1.95 8.12-1.95 5.21 0 9.15 1.71 11.88 5.1 2.76 3.45 4.11 8.4 4.11 14.91v34.6m40.89-14.41v-26.39c-.78-.44-1.77-.75-3.04-1.04-1.29-.29-2.35-.41-3.27-.41-3.76 0-6.62 1.42-8.53 4.22-1.9 2.8-2.86 6.73-2.86 11.77 0 5.32.77 9.16 2.28 11.51 1.51 2.37 3.96 3.55 7.36 3.55 1.39 0 2.78-.29 4.18-.88 1.43-.57 2.72-1.35 3.88-2.33m15.85 14.41h-15.85v-5.55c-1.02.89-2.17 1.82-3.5 2.85-1.29 1.01-2.43 1.77-3.45 2.26-1.25.67-2.51 1.22-3.73 1.57-1.25.36-2.71.55-4.42.55-5.8 0-10.36-2.51-13.73-7.55-3.37-5.05-5.06-11.79-5.06-20.19 0-4.67.58-8.71 1.72-12.13 1.14-3.43 2.73-6.39 4.73-8.94 1.84-2.3 4.07-4.12 6.69-5.5 2.63-1.35 5.36-1.99 8.31-1.99 2.63 0 4.83.29 6.52.88 1.68.57 3.66 1.48 5.91 2.7v-22.76h15.85v73.8zm26.62 1.35c-3.7 0-7.21-.41-10.51-1.27-3.34-.86-6.05-1.87-8.25-3.01v-14.05h1.32c.73.59 1.6 1.25 2.54 1.98.96.69 2.29 1.45 3.98 2.22 1.46.7 3.12 1.32 4.95 1.81 1.83.52 3.81.78 5.96.78 2.19 0 4.14-.34 5.82-1.04 1.68-.7 2.55-1.8 2.55-3.3 0-1.21-.43-2.15-1.18-2.77-.8-.65-2.29-1.25-4.54-1.81-1.19-.32-2.69-.66-4.53-1.01-1.83-.4-3.44-.78-4.82-1.25-3.96-1.29-6.92-3.21-8.9-5.75-1.98-2.54-2.96-5.86-2.96-9.97a15 15 0 011.54-6.69c1.04-2.11 2.57-3.97 4.59-5.6 2.01-1.61 4.48-2.88 7.45-3.84 2.94-.94 6.28-1.4 10-1.4 3.54 0 6.81.37 9.77 1.09 2.97.76 5.48 1.66 7.51 2.7v13.51h-1.25c-.55-.44-1.39-1.02-2.55-1.74-1.17-.73-2.3-1.38-3.42-1.89-1.31-.62-2.74-1.14-4.38-1.53-1.65-.42-3.3-.63-4.97-.63-2.28 0-4.22.39-5.76 1.16-1.57.81-2.36 1.85-2.36 3.14 0 1.19.4 2.15 1.18 2.85.78.68 2.47 1.35 5.06 2.03 1.34.33 2.9.67 4.62 1.01 1.77.34 3.44.78 5.05 1.35 3.61 1.22 6.35 3.03 8.21 5.42 1.89 2.36 2.8 5.52 2.8 9.46 0 2.49-.55 4.87-1.67 7.1-1.1 2.26-2.7 4.15-4.68 5.71-2.19 1.65-4.74 2.96-7.65 3.86-2.91.9-6.44 1.37-10.52 1.37" id="pi-path239" fill="#00553d"/><path d="M188.05 243.46s84.44-17.16 164.72-17.16c118.1 0 140.45 11.56 225.82 18.33 36.62 3.11 67.9 4.49 69.75 4.67 3.56.31 9.34.39 9.34.39v-32.61H188.05v26.38" fill="#f8971d" id="pi-path241"/><path d="M188.05 251.98v18.98h469.63v-18.15s-59.21-2.8-92.88-5.57c-61.05-5.05-96.81-13.53-148.96-14.7-26.36-.57-49.79-2.18-114.33 2.91-12.03.96-56.89 6.25-77.95 10.36-7.71 1.54-35.51 6.17-35.51 6.17" id="pi-path243" fill="#00553d"/></g></g></svg>
```

Comment:
Do we need those id attributes?

- `id="pi-g258"` 
- `id="pi-g245"`
- `id="pi-path239"`
- `id="pi-path241"`
- `id="pi-path243"`
- `id="pi-path5444"`
- `id="pi-path5442"`
- `id="pi-svg5448"`